en-US/main.lang

```diff
@@ -652,3 +652,8 @@ Internet Health
 # Used as title in What's new page for 50
 ;Thanks for choosing Firefox!
 Thanks for choosing Firefox!
+
+
+# HTML page title
```

Comment:
Better: "Used in combination with other strings to create HTML page titles, e.g. in What's new pages"